spec/text-editor-presenter-spec.coffee

```diff
@@ -209,10 +220,36 @@ describe "TextEditorPresenter", ->
         expect(stateFn(presenter).tiles[0]).toBeDefined()
 
     describe "during state retrieval", ->
-      it "does not trigger onDidUpdateState events", ->
+      it "does not trigger ::onDidUpdateState events", ->
         presenter = buildPresenter()
         expectNoStateUpdate presenter, -> presenter.getState()
 
+      it "triggers ::onWillMeasure events before computing any state that needs measurement", ->
```

Comment:
Think I asked this question on another PR with a similar method but... shouldn't this method be called like `onWillNeedMeasurements` or something?